receiver/vcenterreceiver/metadata.yaml

```diff
@@ -120,12 +129,99 @@ attributes:
       - "on"
       - "off"
       - "suspended"
+      - "unknown"
+  entity_status:
+    name_override: status
+    description: The current status of the managed entity.
+    type: string
+    enum:
+      - "red"
+      - "yellow"
+      - "green"
+      - "gray"
   object_name:
     name_override: object
     description: The object on the virtual machine or host that is being reported on.
     type: string
 
 metrics:
+  vcenter.datacenter.cluster.count:
+    enabled: false
+    description: The number of clusters under the datacenter.
+    unit: "{clusters}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation_temporality: cumulative
+    attributes: [entity_status]
+    warnings:
+      if_enabled_not_set: "this metric will be enabled by default starting in release v0.106.0"
+  vcenter.datacenter.host.count:
+    enabled: false
+    description: The number of hosts under the datacenter.
```

Comment:
Does "in" the datacenter make more sense?

Reply:
Yea, this makes sense as well. I can update this for the vms, clusters, and datastores as well.